packages/webapp/src/components/Table/TableV2.jsx

```diff
@@ -184,7 +184,6 @@ export default function TableV2(props) {
                 <TableRow
                   key={row.id || index}
                   onClick={(event) => handleRowClick(event, row)}
-                  isItemSelected={isItemSelected}
```

Comment:
Should this not use the MUI prop `selected={isItemSelected}` ?

https://mui.com/material-ui/api/table-row/